examples/js/couchConfigBasic.js

```diff
@@ -0,0 +1,78 @@
+// These eslint directives prevent the linter from complaining about the use of
+// globals or arguments that will be prevent in CouchDB design doc functions
+// such as views or validate_doc_update
+
+/* global emit, doc, newDoc, oldDoc, userCtx, secObj, sum, rereduce */
+/*eslint no-unused-vars: ["error", { "vars": "local", "argsIgnorePattern": "rereduce|doc|newDoc|oldDoc|userCtx|secObj" }]*/
```

Comment:
I think that a more directed solution to the unused function parameters would be better here, rather than a file-global solution. This solution will apply to all functions and also is dependent on the spelling of the parameters. I think a better solution would be to either remove the unused parameters from the function definitions, or, if we want to keep them for CouchDB common practise reasons, I would use a more focused exception such as:

    // eslint-disable-next-line no-unused-vars

On the line immediately above the function definition.